UpdateMyCount.cls

```diff
@@ -0,0 +1,12 @@
+trigger updateMyCount on Student__c (before insert,before update) {

```

Comment:
can we handle both MyCount and NotAllowingInsertion in single class and trigger

Reply:
ok i changed it